web/client/actions/notifications.js

```diff
@@ -91,6 +91,15 @@ function clear() {
 }
 
 /**
+ * Dispatch a custom action on callback
+ * @memberof actions.notifications
+ * @returns {object}     action
+ */
+function dispatchAction(action) {
```

Comment:
Yes, it's needed because we need to send custom action when the notification button has been toggled